src/components/Card/HorizontalCard/HorizontalCard.jsx

```diff
@@ -0,0 +1,77 @@
+import "./HorizontalCard.css";
+import { FaMinus, FaPlus, FaTrash } from "react-icons/fa";
+import { Button } from "../../index";
+import { useCart } from "../../../contexts";
+export const HorizontalCard = ({ product }) => {
+  const { _id, name, img, description, price, qty } = product;
+  const { cartDispatch, deleteFromCart, updateCartQty } = useCart();
+  const token = localStorage.getItem("UserToken");
+
+  const removeFromCart = async (id) => {
+    try {
+      const deletedItem = await deleteFromCart(id, token);
+      if (deletedItem) {
+        cartDispatch({ type: "REMOVE_FROM_CART", payload: deletedItem });
+      }
+    } catch (err) {
+      console.log(err);
+    }
+  };
+  const updateProductQty = async (id, type) => {
+    try {
+      const updatedCartQty = await updateCartQty(id, type, token);
+      console.log(updatedCartQty);
```

Comment:
remove the console.logs

Reply:
yeah will do that Thanks for pointing it out